backend/api/views/main.py

```diff
@@ -24,26 +24,34 @@ def get_persons():
     return create_response(data={"persons": persons})
 
 
-# function that is called when you visit /census_response
+
+# Given:
+# date in database
+# (optional) tract_id in database
+#
+# Returns all response rates associated to that date (and id)
```

Comment:
Great commenting! It would be great if we could document the params ("date", "tract_id") and the forms we expect them in- an example would be helpful